website/docs/cloud-docs/workspaces/settings/index.mdx

```diff
@@ -122,6 +124,13 @@ By default, new workspaces in Terraform Cloud do not allow other workspaces to a
 
 -> **Note:** The default access permissions for new workspaces in Terraform Cloud changed in April 2021. Workspaces created before this change defaulted to allowing global access within their organization. These workspaces can be changed to more restrictive access at any time. Terraform Enterprise administrators can choose whether new workspaces on their instances default to global access or selective access.
 
+### Drift Detection
+-> Note: Note: Drift detection is in beta and is available in the [Terraform Cloud Business tier](https://www.hashicorp.com/products/terraform/pricing).
+
+When you enable drift detection, Terraform Cloud automatically runs a drift detection assessment for this workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.
+
+Refer to [Drift Detection](/cloud-docs/workspaces/settings/drift-detection) for details.
```

Comment:
We need to add exclusion tags because this feature isn't yet available in TFE :) 

```suggestion
<!-- BEGIN: TFC:only -->
### Drift Detection
-> Note: Note: Drift detection is in beta and is available in the [Terraform Cloud Business tier](https://www.hashicorp.com/products/terraform/pricing).

When you enable drift detection, Terraform Cloud automatically runs a drift detection assessment for this workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.

Refer to [Drift Detection](/cloud-docs/workspaces/settings/drift-detection) for details.
<!-- END: TFC:only -->
```

Reply:
@rexredinger - can you take a look at this? It's on the workspaces --> settings page. Is it still appropriate to say "Drift Detection" here? Just wanted to check in case we need to also change this to "health assessments" or some other term!

Reply:
:+1:
We want to cut over to using "Health Assessments" when talking about the settings that control enabling/disabling drift detection.